### PR TITLE
Created CourseCategory Model and added endpoint for getCourseCategory

### PR DESCRIPTION
**Pull request by timothy-huang**
Defined many-to-one relationship from Course to CourseCategory, and created api endpoint for getCourseCategory

### CODE REVIEW

File: mainframe/settings.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
 # See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get('SECRET_KEY', '')
+SECRET_KEY = os.environ.get('SECRET_KEY', 'SECRET_KEY')
```
Comment (timothy-huang) [inline mainframe/settings.py:27]: Should I change second arg back to an empty string?

Comment (jerrylinew) [reply mainframe/settings.py:27]: nope, let's keep it like this

File: course/models.py [hunk 2]
```diff
@@ -4,29 +4,39 @@
...
+    # Logistical information
+    room = models.CharField(max_length=50)
+    days = models.CharField(max_length=10)
+    schedule = models.TimeField()
+    start_date = models.DateField()
+    end_date = models.DateField()
+    max_capacity = models.IntegerField()
+
+    # Many-to-one relationship with CourseCategory
+    coursecategory = models.ForeignKey(
```
Comment (jerrylinew) [inline course/models.py:41]: nit: let's use underscores to separate words in variables

**Review by jerrylinew — APPROVED** [review]
looks good, just one small nit

**Pull request merged**